release-notes/release-notes542.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: TinyMCE 5.4.2
+title_nav: TinyMCE 5.4.2
+description: Release notes for TinyMCE 5.4.2
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+These release notes provide an overview of the changes for {{site.productname}} 5.4.2, including:
+
+- [TinyMCE 5.4.2 new feature and enhancements](#tinymce542newfeaturesandenhancements)
+- [General bug fixes](#generalbugfixes)
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+> This is the {{site.cloudname}} and {{site.enterpriseversion}} release notes. For information on the latest community version of {{site.productname}}, see: [{{site.productname}} Changelog]({{site.baseurl}}/changelog/).
+
+## TinyMCE 5.4.2 new feature and enhancements
+
+
+
+## General bug fixes
+
+{{site.productname}} 5.4.2 provides fixes for the following bugs:
```

Comment:
There's a couple things we'll have to add here still, but that'll come later :+1: